test/e2e/capv_clusterclass_quickstart_test.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo"
+	"k8s.io/utils/pointer"
+	capi_e2e "sigs.k8s.io/cluster-api/test/e2e"
+)
+
+var _ = Describe("ClusterClass Creation using Cluster API quick-start test [PR-Blocking]", func() {
+	Byf("Creating single-node control plane with one worker node")
+	capi_e2e.QuickStartSpec(context.TODO(), func() capi_e2e.QuickStartSpecInput {
```

Comment:
```suggestion
	capi_e2e.QuickStartSpec(ctx, func() capi_e2e.QuickStartSpecInput {
```
We will eventually replace this with a context which catches the TERM signals to gracefully shut down tests in the face of any custom interruptions.